tests/Php73/Php73Test.php

```diff
@@ -53,6 +53,16 @@ public function testHardwareTimeAsNumType()
 
     public function testHardwareTimeAsNum()
     {
+        // time is not reliable in tests on Windows.
+        // reproduce case:
+        //   $a = 1E6 * (float) microtime(false);
+        //   usleep(1000);
+        //   $b = 1E6 * (float) microtime(false);
+        //   $this->assertGreaterThanOrEqual(1000, $b-$a); // randomly false on windows!
+        if ('\\' === DIRECTORY_SEPARATOR) {
+            $this->markTestSkipped('Test base on time are not reliable on windows');
```

Comment:
`Tests based on time are not reliable on Windows.` (same below)

would `@group time-sensitive` help?